src/components/Pagination.tsx

```diff
@@ -0,0 +1,192 @@
+import React from 'react';
+import styled from 'styled-components';
+import { palette } from "../theme/palette";
+
+export const LinkForPage = styled(({ page, current, href, onClick, className }: {
+  page: number;
+  current?: boolean;
+  href?: string;
+  className?: string;
+  onClick?: (event: React.MouseEvent<HTMLAnchorElement>) => void;
+}) => {
+  const currentValue = current ? "page" : undefined;
+
+  return (
+    <a
```

Comment:
Should we add an aria label for the link? When testing with a screenreader you only hear the numbers, which maybe works because of context, but I'm wondering if something like "Page x" or "Go to page x" would be clearer.